fe/src/components/Icons/BlocksRight.vue

```diff
@@ -0,0 +1,196 @@
+    />
+    <path
+      d="M56.0981 28.4146L31.0844 31.0843L18.9431 12.1413L43.9568 9.47152L56.0981 28.4146Z"
+      fill="#763D22"
+      stroke="#913E16"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M31.0843 31.0843L18.2119 52.6971L6.07064 33.7541L18.9431 12.1413L31.0843 31.0843Z"
+      fill="#763D22"
+      stroke="#913E16"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M43.2256 50.0273L18.212 52.6971L6.07068 33.754L31.0844 31.0843L43.2256 50.0273Z"
+      fill="#9D512D"
+      stroke="#913E16"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M43.2256 50.0274L56.098 28.4145L43.9567 9.47149L31.0843 31.0843L43.2256 50.0274Z"
+      fill="#9D512D"
+      stroke="#913E16"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M43.9568 9.47152L18.9431 12.1413L6.07064 33.7541L31.0843 31.0843L43.9568 9.47152Z"
+      fill="#C46538"
+      stroke="#913E16"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M103 195.5L56 172L9 195.5L56 219L103 195.5Z"
+      fill="#4E2816"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M103 195.5L56 172V125L103 148.5V195.5Z"
+      fill="#894727"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M56 172L9 195.5V148.5L56 125V172Z"
+      fill="#763D22"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M56 219L9 195.5V148.5L56 172V219Z"
+      fill="#C46538"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M56 219L103 195.5V148.5L56 172V219Z"
+      fill="#9D512D"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M103 148.5L56 125L9 148.5L56 172L103 148.5Z"
+      fill="#B05B32"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M200.811 111.373L166.143 71.1432L113.159 74.7392L147.826 114.969L200.811 111.373Z"
+      fill="#4E2816"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M200.811 111.373L166.143 71.1432L184.46 27.317L219.128 67.5472L200.811 111.373Z"
+      fill="#894727"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M166.143 71.1432L113.159 74.7392L131.476 30.913L184.46 27.317L166.143 71.1432Z"
+      fill="#763D22"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M147.826 114.969L113.159 74.7392L131.476 30.913L166.143 71.1432L147.826 114.969Z"
+      fill="#C46538"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M147.826 114.969L200.811 111.373L219.128 67.5472L166.143 71.1432L147.826 114.969Z"
+      fill="#9D512D"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+    <path
+      d="M219.128 67.5472L184.46 27.317L131.476 30.913L166.143 71.1432L219.128 67.5472Z"
+      fill="#B05B32"
+      stroke="#974219"
+      stroke-miterlimit="1"
+    />
+  </svg>
+</template>
+
+<script>
+export default {
+  name: "BlocksRight"
+};
```

Comment:
```suggestion
```
Nuxt has auto import feature based on filename under components directory.
This is no longer needed.
Please remove.

Reply:
Also, please rename components/Icons to component/icons